conf/minio.env

```diff
@@ -1,3 +1,4 @@
-MINIO_MC_CONF_DIR=
-MINIO_ROOT_PASSWORD=
-MINIO_ROOT_USER
+MINIO_MC_CONF_DIR=arlas-data-mc-conf
+MINIO_ROOT_PASSWORD=airssecret
+# at least 8 characters
```

Comment:
which variable is the comment for ? If for password, it should be above it

Reply:
ok